node/utils/metrics/changeTeam.ts

```diff
@@ -0,0 +1,49 @@
+import type { Metric } from './metrics'
+import { sendMetric } from './metrics'
+
+type ChangeTeamFieldsMetric = {
+  date: string
+  user_id: string
+  user_role: string
+  user_email: string
+  org_id: string
+  cost_center_id: string
+}
+
+type ChangeTeamMetric = Metric & { fields: ChangeTeamFieldsMetric }
+
+export type ChangeTeamParams = {
+  account: string
+  userId: string
+  userRole: string
+  userEmail: string
+  orgId: string
+  costCenterId: string
+}
+
+const buildMetric = (metricParams: ChangeTeamParams): ChangeTeamMetric => {
+  return {
+    name: 'b2b-suite-buyerorg-data' as const,
+    account: metricParams.account,
+    kind: 'change-team-graphql-event',
+    description: 'User change team/organization - Graphql',
+    fields: {
+      date: new Date().toISOString(),
+      user_id: metricParams.userId,
+      user_role: metricParams.userRole,
+      user_email: metricParams.userEmail,
+      org_id: metricParams.orgId,
+      cost_center_id: metricParams.costCenterId,
```

Comment:
All those fields are related to the new one. I'll change field names to let them more clearer.
Regarding having old data, at this point, it would be necessary to fetch or receive more data from the client. We are avoiding fetching extra data to not increase the amount of requests and time processing.